### ISSUE TITLE
Relative microcephaly

### ISSUE THREAD

**Issue by SwissGeneticist** [obophenotype/human-phenotype-ontology#10510]
**Preferred term label:**
Relative microcephaly

**Synonyms**
Disproportionately small head - Microcephaly, relative - Relatively small head

**Definition (free text, please give PubMed ID)**
A relatively mild degree of microcephaly in which the head circumference is not below two standard deviations from the mean, but appears dysproportionately small when other factors such as body stature are taken into account.

**Parent term (use hpo.jax.org/app)**
Decreased head circumference HP:0040195

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
# 620086
# 618008
# 614833

**Your nano-attribution (ORCID)**
0000-0001-6762-0819

**Comment by pnrobinson** [comment]
I think this is equivalent to Mild microcephaly HP:0040196, I will relabel that term

**Comment by pnrobinson** [comment]
@SwissGeneticist 
Sorry for the long delay.  The terminology is this area is extremely inconsistent. However, I think that the term "Relative microcephaly is more often defined as follows:

The term “relative microcephaly” is used when an infant is below standard weight and length for gestational age and sex with a proportionally small head circumference measurement. This constellation may be associated with a better intellectual prognosis than “absolute microcephaly” or congenital microcephaly, in which weight and length are normal for gestational age and sex.
PMID: [29150052](https://pubmed.ncbi.nlm.nih.gov/29150052/)

Used in this was, it is synonymous to  Proportional microcephaly

Proportional microcephaly occurs when the head circumference is >2–3 SD below the mean in addition to height and weight being at similar percentiles. While low birth weight and proportional microcephaly has its own set of complications and prognosis, these children do not have the same prognosis and trends of neurocognitive outcomes as those with microcephaly in the setting of normal weight and height.
PMID: 2211965

I think that your definition
A relatively mild degree of microcephaly in which the head circumference is not below two standard deviations from the mean, but appears dysproportionately small when other factors such as body stature are taken into account.

might apply to individuals who height/body weight are ABOVE the mean but they have a "normal" head circumference? In any case, I think it would be confusing to use "Relative microcephaly" for this concept (even though, logically, this is what the English words would imply).

Thoughts?
